src/components/TopMenu.js

```diff
@@ -19,9 +19,9 @@ const TopMenu = ({ topMenu, activeDocMeta }) => {
   return (
     <header>
       <div className="menu">
-        <Link to="/">
+        <a href="/">
```

Comment:
Why did you change this to an anchor link?

Reply:
I changed it because it was just redirecting to the homepage with a hardcoded "/", but since it is still a link between pages, i'm going to change it back to be a gatsby Link.